features/reordering/reordering_placed_order_without_changes.feature

```diff
@@ -36,6 +36,11 @@ Feature: Reordering previously placed order
         When I reorder the order "#00000666"
         Then the "Bank Transfer" payment method should be selected
 
+    @javascript
+    Scenario: Having products taken from previously placed order
```

Comment:
WDYT about moving 
```
Then 2 "Stark Coat" products should be added
```

to 
```
Reordering previously placed order with the same products
```

and removing this scenario?

Reply:
I'm not convinced because then I would have `then` step before `when` step. IMO it's ok as it is now 😃